cmd/lncli/commands.go

```diff
@@ -1520,3 +1520,105 @@ func listChainTxns(ctx *cli.Context) error {
 	printRespJSON(resp)
 	return nil
 }
+
+var signMessageCommand = cli.Command{
+	Name:      "signmessage",
+	Usage:     "sign a message with the node's private key",
+	ArgsUsage: "msg",
+	Description: "Sign msg with the resident node's private key. Returns a the signature as a zbase32 string.\n\n" +
+		"   Positional arguments and flags can be used interchangeably but not at the same time!",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "msg",
+			Usage: "the message to sign",
+		},
+	},
+	Action: signMessage,
+}
+
+func signMessage(ctx *cli.Context) error {
+	ctxb := context.Background()
+	client, cleanUp := getClient(ctx)
+	defer cleanUp()
+
+	var msg []byte
+
+	switch {
+	case ctx.IsSet("msg"):
+		msg = []byte(ctx.String("msg"))
+	case ctx.Args().Present():
+		msg = []byte(ctx.Args().First())
+	default:
+		return fmt.Errorf("msg argument missing")
+	}
+
+	resp, err := client.SignMessage(ctxb, &lnrpc.SignMessageRequest{Msg: msg})
+	if err != nil {
+		return err
+	}
+
+	printRespJSON(resp)
+	return nil
+}
+
+var verifyMessageCommand = cli.Command{
+	Name:      "verifymessage",
+	Usage:     "verify a message signed with the signature",
+	ArgsUsage: "msg signature",
+	Description: "Verify that the message was signed with a properly-formed signature.\n" +
+		"   The signature must be zbase32 encoded and signed with the private key of\n" +
+		"   an active node in the resident node's channel database.\n\n" +
+		"   Positional arguments and flags can be used interchangeably but not at the same time!",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "msg",
+			Usage: "the message to verify",
+		},
+		cli.StringFlag{
+			Name:  "signature",
```

Comment:
Minor nit: I recommend going with simply `sig` here as it's quicker to type on the command line.